sdk/core/azure-perfstress/src/main/java/com/azure/perfstress/SleepTest.java

```diff
@@ -0,0 +1,40 @@
+package com.azure.perfstress;
+
+import reactor.core.publisher.Mono;
+
+import java.time.Duration;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class SleepTest extends PerfStressTest<PerfStressOptions> {
+    private static final AtomicInteger _instanceCount = new AtomicInteger();
+    private final int _secondsPerOperation;
+
+    public SleepTest(PerfStressOptions options) {
+        super(options);
+
+        int instanceCount = _instanceCount.incrementAndGet();
+        _secondsPerOperation = Pow(2, instanceCount);
+    }
+
+    private static int Pow(int value, int exponent) {
+        int power = 1;
+        for (int i=0; i < exponent; i++) {
+            power *= value;
+        }
+        return power;
+    }
```

Comment:
Why not use [Math.pow()](https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#pow-double-double-) method instead of implementing own power method?

Reply:
Yeah, we can use Math.pow(),
unless @mikeharder had any concerns that this might create any performance impact or inconsistencies across languages.

Reply:
I was afraid of the conversion between `double` and `int`.  But if you know this is safe you can replace with `Math.pow()`.

Reply:
Math.pow() works for this usecase, we can replace it